### PR TITLE
Updated to use the latest microservice api versions.

### PR DESCRIPTION
**Pull request by jwomeara**

### CODE REVIEW

**Comment by jwomeara** [comment]
The following PRs are all part of the feature/spring-boot-2.x effort and should be approved/merged together.

https://github.com/NationalSecurityAgency/datawave-accumulo-service/pull/8
https://github.com/NationalSecurityAgency/datawave-audit-service/pull/11
https://github.com/NationalSecurityAgency/datawave-authorization-service/pull/17
https://github.com/NationalSecurityAgency/datawave-config-service/pull/2
https://github.com/NationalSecurityAgency/datawave-dictionary-service/pull/17
https://github.com/NationalSecurityAgency/datawave-hazelcast-service/pull/3
https://github.com/NationalSecurityAgency/datawave-query-metric-service/pull/9
https://github.com/NationalSecurityAgency/datawave-spring-boot-starter-audit/pull/2
https://github.com/NationalSecurityAgency/datawave-spring-boot-starter-cache/pull/1
https://github.com/NationalSecurityAgency/datawave-spring-boot-starter/pull/14
https://github.com/NationalSecurityAgency/datawave-spring-boot-starter-query-metric/pull/3
https://github.com/NationalSecurityAgency/datawave-in-memory-accumulo/pull/11
https://github.com/NationalSecurityAgency/datawave-accumulo-utils/pull/6
https://github.com/NationalSecurityAgency/datawave-base-rest-responses/pull/3
https://github.com/NationalSecurityAgency/datawave-common-utils/pull/2
https://github.com/NationalSecurityAgency/datawave-metadata-utils/pull/18
https://github.com/NationalSecurityAgency/datawave-metrics-reporter/pull/4
https://github.com/NationalSecurityAgency/datawave-type-utils/pull/18
https://github.com/NationalSecurityAgency/datawave-spring-boot-starter-metadata/pull/2
https://github.com/NationalSecurityAgency/datawave/pull/1876

File: pom.xml [hunk 1]
```diff
@@ -78,17 +78,17 @@
...
-        <version.microservice.audit-api>1.4</version.microservice.audit-api>
-        <version.microservice.authorization-api>1.13</version.microservice.authorization-api>
-        <version.microservice.base-rest-responses>1.2</version.microservice.base-rest-responses>
-        <version.microservice.common-utils>1.0</version.microservice.common-utils>
-        <version.microservice.dictionary-api>1.0</version.microservice.dictionary-api>
-        <version.microservice.metadata-utils>1.12</version.microservice.metadata-utils>
-        <version.microservice.metrics-reporter>1.2</version.microservice.metrics-reporter>
-        <version.microservice.query-metric-api>1.5.7</version.microservice.query-metric-api>
-        <version.microservice.type-utils>1.11</version.microservice.type-utils>
+        <version.microservice.accumulo-api>2.0-SNAPSHOT</version.microservice.accumulo-api>
```
Comment (ivakegg) [inline pom.xml:82]: Obviously we will need to change these to non-snapshot versions as soon as we have the other services/utils tagged.

File: web-services/query/src/main/java/datawave/webservice/query/metric/QueryMetricsBean.java [hunk 2]
```diff
@@ -90,22 +90,19 @@ public class QueryMetricsBean {
...
-                // this should keep large queries from blowing up the queue
-                // Leave the last page on the list so that interceptors can update it.
-                Iterator<PageMetric> itr = metric.getPageTimes().iterator();
-                while (metric.getPageTimes().size() > 1) {
-                    itr.next();
-                    itr.remove();
-                }
-            } catch (Exception e) {
-                log.error(e.getMessage(), e);
+        try {
```
Comment (ivakegg) [inline web-services/query/src/main/java/datawave/webservice/query/metric/QueryMetricsBean.java:93]: Why are we getting rid of the hash check?

File: pom.xml (hunk 1 above)
Comment (jwomeara) [reply pom.xml:82]: as you said, this should be obvious.  These versions are not final.  As we talked about yesterday, I need to merge in Keith's accumulo 2.1 changes which will shuffle all of these versions around.  

Also, given that there is a pretty strict ordering to how these various repos need to be built, I will probably handle this when the time comes to start merging the pull requests.  i.e. I will set the final type-utils version when that PR is merged, publish the release, and update each of the remaining PRs to use the published version.  

But for now, these are more or less placeholder versions.  Let's focus on the logic changes for now.

File: web-services/query/src/main/java/datawave/webservice/query/metric/QueryMetricsBean.java (hunk 2 above)
Comment (jwomeara) [reply web-services/query/src/main/java/datawave/webservice/query/metric/QueryMetricsBean.java:93]: @billoley removed that method in the latest query metrics service.  I talked to him about it when I noticed, and this was the change he recommended.

**Review by apmoriarty — DISMISSED** [review]
logic changes look good to me

**Comment by jwomeara** [comment]
> logic changes look good to me

ty.  accumulo 2.1 merge is coming along.  hoping to wrap that up today.

**Review by apmoriarty — APPROVED** [review]
Still LGTM. Great docs.

**Comment by jwomeara** [comment]
Closing this in favor of the spring-boot-2.x-accumulo-2.1 PR.

**Pull request closed without merging**